tests/expr_01.py

```diff
@@ -1,7 +1,8 @@
-def main():
+def main() -> i32:
```

Comment:
Should main return i32 or i64? I actually don't know.

Reply:
@Thirumalai-Shaktivel said that it should return i32, so this should be correct.